recipe/meta.yaml

```diff
@@ -12,28 +13,30 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   entry_points:
     - lomap = lomap.dbmol:startup
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python
+    - python {{ python_min }}
   run:
     - matplotlib-base
     - networkx
     - numpy
-    - python >=3.9
+    - python >={{ python_min }}
     - rdkit
     - gufe >=0.9
+    - netcdf4
```

Comment:
we're getting weird pip check failures, so I added netcdf4 as a temporary measure: see https://github.com/conda-forge/gufe-feedstock/issues/42

Reply:
We could also just remove the `pip check` altogether.